ui/app/common/restclient/services/ReportDAO.js

```diff
@@ -100,6 +100,14 @@
         });
       };
 
+      resource.diffProjectProduct = function (data) {
+        return Configuration.then(function (config) {
```

Comment:
You'll need to rebase against master, the Configuration service has been replaced.